app/services/project_board_cloner.rb

```diff
@@ -84,38 +85,40 @@ def accept_repo_invitation!
   end
 
   def clone_project_board!
-    @message = "Cloning project board."
+    write_message("Cloning project board.")
     @cloned_project_board ||= student_client.create_project(repo_path, project.name)
   end
 
   def enable_issues!
-    @message = "Enabling issues on student repo."
+    write_message("Enabling issues on student repo.")
     student_client.edit_repository(repo_path, has_issues: true)
   end
 
   def update_clone!
-    @message = "Saving GitHub project id in the database."
+    write_message("Saving GitHub project id in the database.")
     clone.update!(github_project_id: cloned_project_board.id, url: cloned_project_board.html_url)
   end
 
   def create_columns!
     column_templates.each.with_index(1) do |column_template, index|
-      @message = "Creating column #{index}."
+      write_message("Creating column #{index} and copy cards.")
       ColumnCloner.run!(column_template, forked_repo, clone.github_project_id, staff_client)
     end
   end
 
   def add_to_dashboard_project!
-    @message = "Adding reference to dashboard project."
+    write_message("Adding reference to dashboard project.")
     staff_client.create_project_card(project.github_column, note: cloned_project_board.html_url)
   end
 
   def email_student!
+    write_message("sending email to student")
     CloneMailer.with(email: email, clone: clone).send_notification.deliver_now
   end
 
-  def write_message!
-    clone.update(message: message)
+  def write_message(message)
+    @clone.update(message: message)
+    @clone.save!
```

Comment:
updated this message writing to update a status for each clone so an admin can see at a glance how things are progressing for a given clone. might be better to number them like "step 5 of 9" or something?